widget/select.go

```diff
@@ -168,9 +218,12 @@ func (s *Select) Tapped(*fyne.PointEvent) {
 		items = append(items, item)
 	}
 
-	s.popUp = newPopUpMenu(fyne.NewMenu("", items...), c)
-	s.popUp.ShowAtPosition(s.popUpPos())
-	s.popUp.Resize(fyne.NewSize(s.Size().Width, s.popUp.MinSize().Height))
+	s.popUp = ShowPopUpMenuAtPosition(fyne.NewMenu("", items...), c, s.popUpPos())
```

Comment:
What was wrong with the previous, existing, API calls?

Reply:
My misreading. I thought the call newPopUpMenu was depreciated. But it was NewPopUpMenu that was so.
I have changed back.